CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ## Unreleased
 
 * Fix travel advice pages to use parent breadcrumbs ([PR #2050](https://github.com/alphagov/govuk_publishing_components/pull/2050))
+* Retire is_page_heading parameter (step 1) ([PR #2051](https://github.com/alphagov/govuk_publishing_components/pull/2051))
```

Comment:
The actual removal of the `is_page_heading` feature will be part of a breaking change release, so I'd rather rephrase this to what it currently does (e.g. Refactor heading logic in radio component)

Reply:
Thanks! 🙏